src/main/player-save.gd

```diff
@@ -65,6 +65,8 @@ const PLAYER_DATA_VERSION := "15d2"
 # filename to use when saving/loading player data. can be changed for tests
 var player_data_filename := "user://turbofat0.save"
 
+var old_save := OldSave.new()
```

Comment:
Document this field, just to make it clear it's for backwards compatibility, and not for a player's previous save file

Reply:
Documented, `Provides backwards compatibility with older save formats`